pkg/controllers/workgenerator/controller_integration_test.go

```diff
@@ -99,20 +105,26 @@ var _ = Describe("Test clusterSchedulingPolicySnapshot Controller", func() {
 			}, timeout, interval).Should(Succeed(), "Failed to get the expected work in hub cluster")
 			By(fmt.Sprintf("work %s is created in %s", work.Name, work.Namespace))
 			// check the binding status
-			wantMC := fleetv1beta1.ResourceBindingStatus{
+			wantC := fleetv1beta1.ResourceBindingStatus{
```

Comment:
```suggestion
			wantStatus := fleetv1beta1.ResourceBindingStatus{
```

hard to understand what's the wantC?